maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -335,12 +340,34 @@ private void installOWAs(Server server, DistroProperties distroProperties) throw
     
     private void downloadOWAs(File targetDirectory, DistroProperties distroProperties, File owasDir) throws MojoExecutionException {
         List<Artifact> owas = distroProperties.getOwaArtifacts(distroHelper, targetDirectory);
-        OpenmrsBintray openmrsBintray = new OpenmrsBintray(getProxyFromSettings());
-
         if (!owas.isEmpty()) {
             wizard.showMessage("Downloading OWAs...\n");
             for (Artifact owa: owas) {
-                openmrsBintray.downloadOWA(owasDir, owa.getArtifactId(), owa.getVersion());
+                wizard.showMessage("Downloading OWA: " + owa);
+                installOwa(owasDir, owa);
+            }
+        }
+    }
+
+    private void installOwa(File owasDir, Artifact owa) throws MojoExecutionException {
```

Comment:
is there reason we have to duplicate these method twice, in BuildDistro and Setup? Or is there a difference I"m not noticing?

Reply:
I didn't want to duplicate it, but didn't really know the right place to put it.  I could add it to "AbstractTask", but that didn't seem to match the model that was followed elsewhere.  I thought about putting it in OwaHelper, but it would have required more wiring around of things.  The same was true for other Helpers and ModuleInstaller.  Ultimately, it was easiest to just duplicate the code, knowing that we'll likely change down the road.  I'll see if I can change this, as I agree it is not ideal to have the same logic duplicated.

Reply:
OK, I pushed an update that moves the new logic into a new method within OwaHelper